### DISCUSSION TITLE
Any clarity on this? Malicious files?

### DISCUSSION THREAD

**Discussion by chiefstobs** [Bing-su/adetailer#563] [General]
See: https://huggingface.co/Bingsu/adetailer/discussions/4

There is a note here on the Files and versions tab that says the following:

"This model has 5 files that have been marked as unsafe.

View unsafe files
person_yolov8n-seg.pt , deepfashion2_yolov8s-seg.pt , person_yolov8s-seg.pt , person_yolov8m-seg.pt , clothing_poor_yolov8s-seg.pt"

Are these files compromised? Or is there something else going on?
![image](https://github.com/Bing-su/adetailer/assets/35372126/3cf3e382-c66e-4c8c-a731-6b989c9e4163)

**Comment by askAvoid** [comment]
Really, no response from the author? I think this says it all. I hope users her have good firewall software.

**Comment by thiagojramos** [comment]
I saw that. But I suspect it might be because the file extensions are `.pt` (**P**ickle**T**ensor), which from what I read are file extensions supported by PyTorch. Here's a discussion on Reddit that has some explanations about why this type of file is unsafe, and **CAN** (≠ **HAVE**) so-called "malicious" code:

https://www.reddit.com/r/StableDiffusion/comments/117404t/are_pt_and_safetensor_files_actually_secure/

And if I remember correctly, it was a discussion like this that made people use/create models (LoRa/Checkpoint) for SD with the `.safetensor` extension (and I believe the "safe" in the name is precisely for this context). Before, it was something like `.ckpt` ("ck" I think comes from "checkpoint", and "pt" from "pickletensor"). But as I said, it's just my theory. Someone who understands better should correct or explain it better to me.
